data-browser/src/test/scala/hmda/dataBrowser/DataBrowserIntegrationSpec.scala

```diff
@@ -12,11 +12,13 @@ import hmda.dataBrowser.services.{ DataBrowserQueryService, HealthCheckService,
 import hmda.utils.EmbeddedPostgres
 import monix.eval.Task
 import org.scalamock.scalatest.MockFactory
-import org.scalatest.{ Matchers, WordSpec }
+import org.scalatest.{ Matchers, WordSpec, Tag }
 import org.slf4j.LoggerFactory
 import slick.basic.DatabaseConfig
 import slick.jdbc.JdbcProfile
 
+object CustomTag extends Tag("actions-ignore")
```

Comment:
Here are the tags that let us exclude the embed tests.

Reply:
Recommend a name like "IgnoreTag" or something more descriptive.